test/test_viscous.py

```diff
@@ -240,6 +240,45 @@ def test_diffusive_heat_flux(actx_factory):
         assert discr.norm(j[ispec+1] - exact_j, np.inf) < tol
 
 
+@pytest.mark.parametrize("dim", [1, 2, 3])
+def test_local_max_species_diffusivity(actx_factory, dim):
+    actx = actx_factory()
+    nel_1d = 5
+
+    from meshmode.mesh.generation import generate_regular_rect_mesh
+
+    mesh = generate_regular_rect_mesh(
+        a=(1.0,) * dim, b=(2.0,) * dim, n=(nel_1d,) * dim
```

Comment:
```suggestion
        a=(1.0,) * dim, b=(2.0,) * dim, nelements_per_axis=(nel_1d,) * dim
```
(`n` is deprecated.)